package.json

```diff
@@ -388,7 +388,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "FAST_LINT=true lint-staged"
+      "pre-commit": "lint-staged"
```

Comment:
The `FAST_LINT` is intentional because it's soooo much slower. It disables type information for linting. The idea was, that small changes, shouldn't make you wait so long for the linting when you commit. Obviously it makes sense when you change a lot of things, so it was a trade-off decision that favors convenience and speed over thorough linting, which still can be achieved by manually running `npm run lint`. Up to you :-)